src/VisualStudio/Core/Def/Telemetry/VisualStudioSyntaxTreeConfigurationService.cs

```diff
@@ -39,8 +43,20 @@ internal sealed class OptionsMetadata
                 new FeatureFlagStorageLocation("Roslyn.DisableRecoverableTrees"));
 
             public static readonly Option2<bool> DisableProjectCacheService = new(
-                "WorkspaceConfigurationOptions", "DisableProjectCacheService", defaultValue: false,
+                "WorkspaceConfigurationOptions", nameof(DisableProjectCacheService), defaultValue: false,
```

Comment:
I am always unsure if we should use nameof or a hardcoded string for this specific argument to passed to Option constructor. Using nameof is cleaner, but renaming this field will lead to change in this argument value and hence ignoring the existing user specific value (if the option has already shipped), which is likely not what we want. I see some parts of our code base use nameof and some parts an explicitly specified string. @tmat do we have a standard approach we want to follow throughout the repo?

Reply:
➡️ I've had to resolve merge conflicts on this 5 times already. This string is semantically fine so leaving it.